packages/zowe-explorer-ftp-extension/src/ftpSession.ts

```diff
@@ -9,17 +9,18 @@
  *
  */
 
+import { zosNodeAccessor } from "@zowe/zos-ftp-for-zowe-cli";
 import { imperative } from "@zowe/zowe-explorer-api";
 
 export class FtpSession extends imperative.Session {
-    public ussListConnection;
-    public mvsListConnection;
-    public jesListConnection;
+    public ussListConnection: zosNodeAccessor.ZosAccessor;
+    public mvsListConnection: zosNodeAccessor.ZosAccessor;
+    public jesListConnection: zosNodeAccessor.ZosAccessor;
     public constructor(newSession: imperative.ISession) {
         super(newSession);
     }
 
-    /* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access*/
+    /* eslint-disable zowe-explorer/no-floating-promises*/
```

Comment:
Instead of disabling a lint rule, can we explicitly ignore the promises like this?
`void this.ussListConnection.close();`